packages/hint/src/lib/config.ts

```diff
@@ -213,10 +213,13 @@ export class Configuration {
     }
 
     /**
+     * @deprecated
      * Generates the list of browsers to target using the `browserslist` property
      * of the `hint` configuration or `package.json` or uses the default one
      */
     public static loadBrowsersList(config: UserConfig) {
+        console.warn('`Configuration.loadBrowsersList` is deprecated. Use `Configuration.fromConfig` instead.');
```

Comment:
Do we want to use our [logger](https://github.com/webhintio/hint/blob/master/packages/hint/src/lib/utils/logging.ts) for this or is it a bit overkill?

Reply:
Looks like `logger` currently only has `log` and `error`. Do you want to me to add a `warn` or just use `error` here?

Reply:
Ok, I updated the logger to include a `warn` and used that.